client/src/components/landing.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState, useEffect } from 'react';
+import Player from './player.jsx';
+import Login from './login.jsx';
+import './App.css';
+
+function Landing() {
+  const [token, setToken] = useState('');
+
+  useEffect(() => {
+    async function getToken() {
+      const response = await fetch('/auth/token');
+      const json = await response.json();
+      setToken(json.access_token);
+    }
+
+    getToken();
+  }, []);
+
+  return (
+    <>
+      { (token === '') ? <Login/> : <Player token={token} />}
```

Comment:
Your conditional rendering of this using a ternary  operator is clever and looks nice here. I commend you on this.